### PR TITLE
fix(snapshot/grpc): add missing snapshot state conversions

### PR DESCRIPTION
**Pull request by tiagolobocastro**

### CODE REVIEW

File: control-plane/stor-port/src/types/v0/store/snapshots/replica.rs [hunk 1]
```diff
@@ -117,6 +121,14 @@ impl ReplicaSnapshotMeta {
     pub fn size(&self) -> u64 {
         self.size
     }
+    /// Get the snapshot size.
```
Comment (dsharma-dc) [inline control-plane/stor-port/src/types/v0/store/snapshots/replica.rs]: "/// Get the snapshot timestamp."

File: control-plane/stor-port/src/types/v0/transport/replica.rs [hunk 2]
```diff
@@ -174,10 +178,30 @@ impl ReplicaSnapshotDescr {
         &self.snap_uuid
     }
 
+    /// Get a reference to snapshot id.
```
Comment (dsharma-dc) [inline control-plane/stor-port/src/types/v0/transport/replica.rs]: "/// Get a reference to pool id."

**Review by dsharma-dc — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

File: control-plane/stor-port/src/types/v0/store/snapshots/replica.rs (hunk 1 above)
Comment (tiagolobocastro) [reply control-plane/stor-port/src/types/v0/store/snapshots/replica.rs]: Thanks, there were a few more too, fixed all I think.

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**